temoa/core/config.py

```
@@ -153,7 +153,7 @@ def __init__(
                     SE.write('Warning: ' + msg)
 
     @staticmethod
-    def build_config(config_file: Path, output_path: Path, silent=False) -> 'TemoaConfig':
+    def build_config(config_file: Path, output_path: Path, silent: bool = False) -> 'TemoaConfig':
```

Ruff (0.14.0)

Boolean-typed positional argument in function definition

(FBT001)